### ISSUE TITLE
Sanchit comments

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/calendar#47]
https://github.com/GroceriStar/react-only-intern-23/issues/70#issuecomment-406884595

**Comment by atherdon** [comment]
@sanchit94 if you wish - we can discuss here a tasks, that should be done by other React interns, in order to make code of this repository more clear...

**Comment by sanchit94** [comment]
Hey Arthur, I think a few things can be done to make this repo's code better:
1. The function checks for a new task every 15 mins, that should be changed to check every minute. This would be good practice.
2. Responsiveness of the calender component.
3. Also, if someone can write this code with better algo, he/she must be encouraged.

**Comment by atherdon** [comment]
Yeah, i think your code is a good starting point, but it should be
improved, because we're trying to put more functionality into it.
Thank you for our comments. Tell me if you find something else
Btw, i also think that we should start to use forms from your #23
repository - because code looks good at that files...

**Comment by sanchit94** [comment]
Yeah react-final-form code is really great, I think it should be used for the form part.

**Comment by sanchit94** [comment]
I want to continue with this project on react but due to shortage of time I am not able to get my head into it. Additionally I am working on  project in Angular so I have much better perspective of these frameworks now. However, once I am done with the Angular project I would love to contribute to this project as I have not written JAVASCRIPT for a while!

**Comment by atherdon** [comment]
cool! this is a perfect news! we actually re-forming our powers and starting to work on main GS React frontend
